plugins/org.polarsys.capella.scenario.editor.dsl/org.polarsys.capella.scenario.editor.dslscenario.ui/xtend-gen/org/polarsys/capella/scenario/editor/dslscenario/ui/contentassist/DslProposalProvider.java

```diff
@@ -95,18 +94,46 @@ public void getExistingParticipants(final String keyword, final ContentAssistCon
   public void completeSequenceMessage_Source(final EObject model, final Assignment assignment, final ContentAssistContext context, final ICompletionProposalAcceptor acceptor) {
     EList<Participant> _variablesDefinedBefore2 = this.variablesDefinedBefore2(((Model) model));
     for (final EObject el : _variablesDefinedBefore2) {
-      acceptor.accept(this.createCompletionProposal(((Actor) el).getName(), ((Actor) el).getName(), null, context));
+      String _name = ((Participant) el).getName();
+      String _plus = ("\"" + _name);
+      String _plus_1 = (_plus + "\"");
+      String _name_1 = ((Participant) el).getName();
+      String _plus_2 = ("\"" + _name_1);
+      String _plus_3 = (_plus_2 + "\"");
+      acceptor.accept(
+        this.createCompletionProposal(_plus_1, _plus_3, 
+          null, context));
     }
   }
   
   @Override
   public void completeSequenceMessage_Target(final EObject model, final Assignment assignment, final ContentAssistContext context, final ICompletionProposalAcceptor acceptor) {
     EList<Participant> _variablesDefinedBefore3 = this.variablesDefinedBefore3(((SequenceMessage) model));
     for (final EObject el : _variablesDefinedBefore3) {
-      acceptor.accept(this.createCompletionProposal(((Actor) el).getName(), ((Actor) el).getName(), null, context));
+      String _name = ((Participant) el).getName();
+      String _plus = ("\"" + _name);
+      String _plus_1 = (_plus + "\"");
+      String _name_1 = ((Participant) el).getName();
+      String _plus_2 = ("\"" + _name_1);
+      String _plus_3 = (_plus_2 + "\"");
+      acceptor.accept(
+        this.createCompletionProposal(_plus_1, _plus_3, 
+          null, context));
```

Comment:
can also update this version with _plus_x variables and do as for participants instead.